CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @pbakker @kilink @srinivasankavitha @iuliiasobolevska @kzwang @asibross @iparadiso @jjacobs44
```

Comment:
```suggestion
* @paulbakker @kilink @srinivasankavitha @iuliiasobolevska @kzwang @asibross @iparadiso @jjacobs44
```

Reply:
Thanks for the catch @setchy. You would think I at least get my own username right, but no 🥹

Reply:
🤣 all good mate